alf/utils/summary_utils.py

```diff
@@ -327,6 +347,19 @@ def _summarize_one(path, dist):
             else:
                 return
 
+            if ind is not None:
```

Comment:
Would be great to add a one line comment saying this is for handling the mixture distribution, picking the component with highest weight.